contrib/examples/apiserver/apiserver.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: service-catalog-apiserver
+  labels:
+    app: apiserver
+spec:
+  containers:
+  - name: apiserver
+    image: apiserver
+    args: 
+    - --etcd-servers
+    - http://localhost:2379
+    - -v
+    - "10"
+    ports:
+    - name: https
+      containerPort: 6443
+  - name: etcd
+    image: gcr.io/google_containers/etcd:3.0.14
```

Comment:
@pmorie I think that's what @luksa is asking. I was saying what we did have using. The quay image appears to be 3.1.0